### ISSUE TITLE
Encoding character bug in .from_string

### ISSUE THREAD

**Issue by Peticali** [jarrekk/imgkit#69]
When using .from_string function, characters like °,Ç,ã, etc, has encoding bugs

**Reproducing**
My html string looks like this: "objeto aguardando retirada no endereço indicado Para retir**á**-lo, é preciso informar o c**ó**digo do objeto".

**So i created this code for testing:**
![Screenshot_3](https://user-images.githubusercontent.com/34588988/114247504-9f69e600-996b-11eb-81d1-d1f5445d5659.png)

**Outputs:**

### _-With .from_file:_
![HtmlFromFile](https://user-images.githubusercontent.com/34588988/114247650-fc659c00-996b-11eb-9833-2d006ff261cb.jpg)

### _-With .from_string:_
![HtmlFromString](https://user-images.githubusercontent.com/34588988/114247664-05ef0400-996c-11eb-99de-18361a512ed7.jpg)

 - OS: Windows 10 20H2
 - IMGkit Version: 1.2.1
- `wkhtmltopdf` Version: 0.12.6 (with patched qt)

**Comment by jarrekk** [comment]
@Peticali Thanks for the information!
IMGkit didn't take these characters into consideration in the previous, as all strings are converted to bytes via `utf-8` charset and pipe in `wkhtmltoimage`. If the string has `<meta charset="UTF-8">`, there is no encoding issues:

``` python
import imgkit
imgkit.from_string('<meta charset="UTF-8">°,Ç,ã', 'out.jpg')
```
I will figure out a way to fix it in the next version.

**Comment by jarrekk** [comment]
Fixed in [1.2.2](https://pypi.org/project/imgkit/1.2.2/).
